package.json

```diff
@@ -34,6 +34,7 @@
     "react": "18.2.0",
     "react-i18next": "^13.0.1",
     "react-native": "0.72.1",
+    "react-native-alert-notification": "^0.4.2",
```

Comment:
We are trying to build our own lib, and hence would like to avoid taking a dependency on third party libs.

We should use [alert from RN lib](https://reactnative.dev/docs/alert), and customize as per our need.